modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -17,32 +17,32 @@
     header("HTTP/1.1 403 Forbidden");
     exit;
 }
-
 set_include_path(
+
     get_include_path().
-    ":../../project/libraries:../../php/libraries:"
+    ':../../project/libraries:../../php/libraries:'
 );
-require_once "NDB_Client.class.inc";
-require_once "NDB_Config.class.inc";
-require_once "Email.class.inc";
-$client = new NDB_Client();
-$client->initialize("../../project/config.xml");
+require_once 'NDB_Client.class.inc';
+require_once 'NDB_Config.class.inc';
+require_once 'Email.class.inc';
 $factory = NDB_Factory::singleton();
 $baseURL = $factory->settings()->getBaseURL();
 
 $config = NDB_Config::singleton();
+$base = $config->getSetting('base');
+$name = $userSingleton->getUsername();
 
 // create Database object
 $DB =& Database::singleton();
 
 $editNotifier = new NDB_Notifier(
-    "document_repository",
-    "edit"
+    'document_repository',
+    'edit'
 );
 
 $uploadNotifier = new NDB_Notifier(
-    "document_repository",
-    "upload"
+    'document_repository',
+    'upload'
```

Comment:
YES. Take those unnecessary double quotes down!!

Reply:
Definitely a small change but in some cases using single quotes vs. double quotes can [give a small boost in speed](https://stackoverflow.com/questions/3446216/what-is-the-difference-between-single-quoted-and-double-quoted-strings-in-php?answertab=votes#tab-top). It kind of depends on what the compiler does ultimately and probably doens't matter but.... figured I might as well at least keep this file consistent, one way or another.